src/praisonai-code/tests/unit/test_interactive_clarify_tool.py

```
@@ -0,0 +1,164 @@
+from praisonai_code.cli.features import interactive_tools as it
+from praisonai_code.cli.features.interactive_tools import (
+    TOOL_GROUPS,
+    ToolConfig,
+    get_interactive_tools,
+    resolve_tool_groups,
+)
+
+_clarify_available = True
+try:  # pragma: no cover - environment dependent
+    import praisonaiagents.tools.clarify  # noqa: F401
+except Exception:  # pragma: no cover
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)